src/editors/containers/TextEditor/index.jsx

```diff
@@ -97,7 +98,29 @@ const TextEditor = ({
                 screenreadertext={intl.formatMessage(messages.spinnerScreenReaderText)}
               />
             </div>
-          ) : (selectEditor())}
+          ) : (
+            <>
+              <TextEditorPluginSlot
+                updateContent={(newContent) => {
+                  // Update the editor
```

Comment:
It seems to me that, based on the `updateContent` prop name, it’s already clear that this is responsible for updating the content. So let’s remove this comment.